src/main/webapp/trips/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>TravIS Trips</title>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" 
+      integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    <link rel="stylesheet" href="../style.css">
+    <script src="../script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <div id="header">
+        <h3 class="header-child float-left" id="header-title">TravIS</h3>
+        <form class="header-child float-right" id="sign-out-button-form" action="/auth" method="POST">
+          <input class="btn btn-primary" type="submit" value="Sign out">
+        </form>
+        <div class="header-child float-right">
+          <a class="btn btn-primary" href="/trips/">Trips</a>
+        </div>
+      </div>
+      <div id="index-registration-block">
+        <h1 class="index-registration-block-child">TravIS: Travel Information System</h1>
+        <p class="index-registration-block-child" id="index-registration-block-p">
+          TravIS, your personal Travel Information System, allows you to plan 	
+          the optimal trip to wherever you want to visit.	
+        </p>	
+        <form action="/auth" method="POST" class="index-registration-block-child"	
+          id="sign-in-button-form">	
```

Comment:
I think this page will probably change to be different from index.html fairly soon. But if not, or if those pages will have some parts that are identical long-term, then you might consider looking for js libraries that would let you factor out the common parts (eg: handlebars.js i think? there may be others). Definitely not something to worry about on this CL though.